com.alizadicomviewer.AlizaMS.metainfo.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id type="desktop">com.alizadicomviewer.AlizaMS.desktop</id>
+
+  <name>Aliza MS</name>
+  <summary>A DICOM viewer</summary>
+
+  <metadata_license>FSFAP</metadata_license>
+  <project_license>GPL-3.0-only</project_license>
+
+  <description>
+    <p>
+      A DICOM viewer.
+    </p>
+    <p>See DICOM storage classes for read support,
+      3D view for uniform and non-uniform images,
+      De-identify DICOM,
+      Ultrasound incl. proper measurement in regions, cine,
+      Grayscale Softcopy Presentation,
+      Structured Report,
+      Siemens Mosaic format,
+      United Imaging Healthcare (UIH) Grid / VFrame format,
+      Sequential and interleaved 4D series,
+      Parametric Map,
+      Supplemental Palette Color Lookup,
+      Elscint ELSCINT1 PMSCT_RLE1 and PMSCT_RGB1,
+      MR Spectroscopy (spatial and data information).
+    </p>
+    <p>
+      Recursive scan directory,
+      DICOMDIR,
+      CTK database.
+    </p>
+    <p>
+      DICOM metadata viewer 2D and 3D Views,
+      Image intensity windowing, linear and sigmoid, LUT,
+      MPR (multi-planar reconstruction),
+      Physical space,
+      Non-uniform DICOM series,
+      Contours,
+      Multiple images,
+      Volume rendering.
+    </p>
+    <p>
+      Image intensity windowing, LUT,
+      Image intensity windowing with histogram,
+      Select sub-image,
+      Set position with cursor (MPR),
+      Measure,
+      Ultrasound regions,
+      View value and index under cursor,
+      3D animation (2D+time),
+      4D animation (3D+time),
+      Anchor slice.
+    </p>
+  </description>
+
+  <url type="homepage">https://github.com/AlizaMedicalImaging/AlizaMS</url>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://raw.githubusercontent.com/AlizaMedicalImaging/AlizaMS/master/package/art/alizams_scr2.jpg</image>
```

Comment:
by linking to master you risk having the package broken when things change. You should link to a specific tag or revision that will be immutable.